core/geometry/src/test/geometry4d/MomentData.test.ts

```diff
@@ -202,7 +202,7 @@ describe("MomentData", () => {
         GeometryCoreTestIO.showMomentData(allGeometry, rawSums, true, x0, y0, 0);
         // GeometryCoreTestIO.showMomentData(allGeometry, principalMoments, true, x0, y0, 0);
         if (g instanceof CurveCollection) {
-          const strokes = g.cloneStroked(strokeOptions);
+          const strokes = (g as CurveCollection).cloneStroked(strokeOptions);

```

Comment:
@EarlinLutz tsc was basically complaining that `g` could never be a `CurveCollection` before I added this type assertion... Is this condition not even hittable?  If it is, we may need to remove the type assertion and investigate why tsc thinks this is impossible.

Reply:
`g` is always a `CurvePrimitive` here - compiler is correct.

Reply:
Thanks Paul - I'll remove this.